clm/mlperf_logging_utils.py

```diff
@@ -126,7 +136,7 @@ def on_train_begin(self, args, state, control, **kwargs):
         self.mllogger.event(
             key=constants.OPT_GRADIENT_CLIP_NORM, value=args.max_grad_norm
         )
-        self.mllogger.event(key=constants.OPT_BASE_LR, value=args.lr)
+        self.mllogger.event(key=constants.OPT_BASE_LR, value=args.learning_rate)
```

Comment:
Conflict in torch xla side

In torch xla we parse hydra config as args and there's only `lr` in hydra config.

Reply:
This can be changed to lr then